python_modules/dagster/dagster/_cli/asset.py

```diff
@@ -111,7 +112,46 @@ def asset_list_command(**kwargs):
     asset_keys = asset_selection.resolve(repo_def.asset_graph, allow_missing=True)
 
     for asset_key in sorted(asset_keys):
-        print(asset_key.to_user_string())  # noqa: T201
+        print(asset_key.to_user_string())
+
+
+@asset_cli.command(name="report", help="Report asset materialization")
+@python_origin_target_argument
+@click.option("--select", help="Asset selection to target", required=True)
```

Comment:
"asset selection" normally implies an expression that can refer to multiple assets. We just want this to apply to a single asset, right? In that case, we should use `--key`, like in the wipe command.